burotel/indico_burotel/client/js/components/Stats.jsx

```diff
@@ -0,0 +1,163 @@
+  const color = interpolateRGB(PALETTE[1], PALETTE[0], ratio);
+  return (
+    <Table.Cell
+      style={{color: toHex(color)}}
+      styleName={highlight ? 'cell-highlight' : 'cell-normal'}
+    >
+      {`${toPercent(ratio)}%`}
+      <div className="percentage">{value}</div>
+    </Table.Cell>
+  );
+}
+
+PercentCell.propTypes = {
+  value: PropTypes.number.isRequired,
+  total: PropTypes.number.isRequired,
+  highlight: PropTypes.bool,
+};
+
+PercentCell.defaultProps = {
+  highlight: false,
+};
+
+function StatsTable({data, numDays, months}) {
+  return (
+    <div>
+      <div styleName="stats-chart">
+        <Chart data={calculateChartData(data, months)} axes={AXES} tooltip dark />
+      </div>
+      <Table striped>
+        <Table.Header>
+          <Table.Row>
+            <Table.HeaderCell>
+              <Translate>Building</Translate>
+            </Table.HeaderCell>
+            <Table.HeaderCell>
+              <Translate>Experiment</Translate>
+            </Table.HeaderCell>
+            <Table.HeaderCell>
+              <Translate>Number of desks</Translate>
+            </Table.HeaderCell>
+            {months.map(({name}) => (
+              <Table.HeaderCell key={name}>{name}</Table.HeaderCell>
+            ))}
+            <Table.HeaderCell>
+              <Translate>Total</Translate>
+            </Table.HeaderCell>
+          </Table.Row>
+        </Table.Header>
+        <Table.Body>
+          {data.map(([building, experiments]) =>
+            experiments.map(([experiment, {bookings, deskCount, months: monthData}]) => (
+              <Table.Row key={`row-${building}-${experiment}`}>
+                <Table.Cell>{building}</Table.Cell>
+                <Table.Cell>{experiment}</Table.Cell>
+                <Table.Cell>{deskCount}</Table.Cell>
+                {monthData.map((value, i) => (
+                  <PercentCell
+                    key={`month-${months[i].id}`}
+                    value={value}
+                    total={deskCount * months[i].numDays}
+                  />
+                ))}
+                <PercentCell value={bookings} total={deskCount * numDays} highlight />
+              </Table.Row>
+            ))
+          )}
+        </Table.Body>
+      </Table>
+    </div>
+  );
+}
+
+StatsTable.propTypes = {
+  numDays: PropTypes.number.isRequired,
+  data: PropTypes.arrayOf(PropTypes.array).isRequired,
+  months: PropTypes.arrayOf(PropTypes.object).isRequired,
+};
+
+export default function Stats({startDate, endDate}) {
+  const [loading, setLoading] = useState(true);
+  const [data, setData] = useState(null);
+
+  useEffect(() => {
+    (async () => {
+      setLoading(true);
+      const {data} = await indicoAxios.get(getBurotelStats(), {
+        params: {
+          start_month: startDate.format('YYYY-M'),
+          end_month: endDate.format('YYYY-M'),
+        },
+      });
+      setData(data);
+      setLoading(false);
+    })();
+  }, [startDate, endDate]);
```

Comment:
I think you could use `useIndicoAxios` for this with `trigger: [startDate, endDate]`